cli/src/main/java/com/devonfw/tools/ide/tool/mvn/Mvn.java

```diff
@@ -29,4 +32,35 @@ public boolean install(boolean silent) {
     return super.install(silent);
   }
 
+  @Override
+  protected void postInstall() {
+
+    for (PluginDescriptor plugin : super.getConfiguredPlugins()) {
+
+      if (plugin.isActive()) {
```

Comment:
This was all already implemented in the super-class.
The only thing a plugin based tool needs to implement is the `installPlugin` method.
IMHO something is going wrong here and before we discuss this PR/Story (mvn) we should complete and merge #142